### PR TITLE
Add French transcreated content for edit PDF feature

### PR DESCRIPTION
**Pull request by stephaniehobson**
## One-line summary

Add French transcreated content for edit PDF feature

## Significant changes and points to review

- Created French variant of features/pdf-editor
- Added French features/pdf-free
- Added French page features/pdf-comp

## Issue / Bugzilla link

#15098

## Testing

http://localhost:8000/fr/firefox/features/pdf-editor/
http://localhost:8000/fr/firefox/features/free-pdf-editor/
http://localhost:8000/fr/firefox/features/complete-pdf/

### CODE REVIEW

File: bedrock/firefox/templates/firefox/features/pdf-editor-fr.html [hunk 1]
```diff
@@ -0,0 +1,105 @@
...
+
+<p>Avec <a href="{{ url('firefox.features.pdf-free') }}">l’éditeur PDF</a> Firefox, fini de se retrouver sur des interfaces peu sécurisées, chargées de pubs ou encore télécharger des logiciels encombrants. </p>
+
+<p><a href="{{ url('firefox.features.pdf-complete') }}">Remplir un PDF</a>, le modifier ou l'enregistrer n’a jamais été aussi simple et rapide. Qu'il s'agisse d'une déclaration d’impôts, d’un contrat à signer ou de documents de travail, il suffit de quelques clics pour remplir un PDF&nbsp;!</p>
+
+<h2>Modifier un PDF en ligne rapidement et facilement</h2>
+
+<ol class="mzp-u-list-styled">
+  <li>
+    <p>Il suffit d’ouvrir le PDF à remplir directement dans Firefox. Retrouver votre PDF sur votre ordinateur, de faire un clic droit dessus et de sélectionner "Ouvrir avec Firefox".</p>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-editor-fr.html:30]: **Nit:** It seems you're combining point 1 and point 2 from the copy into this one list item?

Screenshot 2024-10-09 at 10 02 24 AM

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+ This Source Code Form is subject to the terms of the Mozilla Public
+ License, v. 2.0. If a copy of the MPL was not distributed with this
+ file, You can obtain one at https://mozilla.org/MPL/2.0/.
+#}
+
+{% extends "firefox/features/base-article.html" %}
+
+
+{% block article_title_short %}Remplir un PDF en ligne: fiche d’impôts{% endblock %}
+{% block article_title %}Remplir un PDF en ligne : fiche d’impôts{% endblock %}
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:11]: **Nit:** Based on the copy, these titles should be `Comment remplir sa fiche d’impôts en ligne via PDF? `?

File: bedrock/firefox/templates/firefox/features/pdf-editor-fr.html [hunk 3]
```diff
@@ -0,0 +1,105 @@
...
+    </figure></p>
+  </li>
+</ol>
+
+<p>Modifier des PDF oui, mais de façon simple et sécurisée !</p>
+
+<hr>
+
+<h2>Vous n'avez pas encore Firefox ?</h2>
+<p><a href="{{ url('firefox.features.index') }}">En savoir plus</a> sur nos fonctionnalités, ou <a href="">Téléchargez Firefox</a> dès maintenant.</p>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-editor-fr.html:102]: **Issue (blocking):** The second anchor tag in this line is missing an HREF that should lead to the Download Firefox page? But also, should said link only show up on non-firefox browsers? We already have a Download Firefox CTA for non-firefox browsers:

Screenshot 2024-10-09 at 10 53 59 AM

File: bedrock/firefox/templates/firefox/features/pdf-free-fr.html [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </p>
+  </li>
+  <li>
+    <p><strong>Signer :</strong> Pour signer au bas de la demande, cliquez sur l'icône Dessin et ajoutez votre signature.</p>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-free-fr.html:71]: **Nit:** Based on copy, there should be an image under this list item?

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html [hunk 5]
```diff
@@ -0,0 +1,94 @@
...
+{% block page_desc %}Retrouvez un guide pas-à-pas pour remplir et même corriger sa déclaration d’impôts, en utilisant les options intégrées de Firefox.{% endblock %}
+
+{% block article_content %}
+
+<p>Fini les soucis liés à la paperasse, place à la simplicité administrative !  Que vous ayez besoin de remplir une première déclaration, de corriger une erreur ou de gérer vos justificatifs, Firefox vous offre une solution rapide et sécurisée pour vos données. </p>
+
+<p>Remplir des PDF n'a jamais été aussi simple et fiable, même quand on parle de fiches d’impôts.</p>
+
+<h2>Remplir sa PDF</h2>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:20]: **Nit:** Based on copy, this heading should say `Remplir sa déclaration de revenus`?

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html [hunk 6]
```diff
@@ -0,0 +1,94 @@
...
+        },
+        optional_attributes={
+          "alt": "",
+          "width": "600"
+        }
+      ) }}
+    </p>
+  </li>
+  <li>
+    <p>Vous pouvez déplacer n’importe quel de vos blocs de texte et ajuster la taille, comme bon vous semble. Vous pouvez facilement passer d’une page à l’autre de votre déclaration ou avoir une vue d’ensemble en sélectionnant dans les Outils. </p>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:59]: **Nit:** (again based on copy lol) it doesn't seem this should be a list item?

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html [hunk 7]
```diff
@@ -0,0 +1,94 @@
...
+<p>L’éditeur PDF Firefox vous aide à remplir mais aussi à classer et archiver vos tâches administratives, en toute confidentialité.</p>
+
+<p>Vous pouvez organiser vos justificatifs comme les factures, les reçus de dons, et l’imprimé fiscal unique. Ces documents n'ont pas à être fournis lors de la déclaration mais doivent être conservés en cas de contrôle, pendant au moins 3 ans. Pourquoi ne pas se simplifier la tâche dès l’étape de la déclaration&nbsp;? </p>
+
+<p>Enregistrez tous vos PDF dans des dossiers bien nommés pour une gestion facile et sécurisée.</p>
+
+<hr>
+
+<h2>Vous n'avez pas encore Firefox ?</h2>
+<p><a href="{{ url('firefox.features.index') }}">En savoir plus</a> sur nos fonctionnalités, ou <a href="">Téléchargez Firefox</a> dès maintenant.</p>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:91]: **Issue (blocking)** second anchor tag in this line is missing an HREF to the download firefox page

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:1]: **Question: Do we know where this new page will be linked from?**

File: bedrock/firefox/templates/firefox/features/pdf-free-fr.html
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-free-fr.html:1]: **Question: Do we know where this new page will be linked from?**

**Review by reemhamz — CHANGES_REQUESTED** [review]
A few blocking nits and issues that shouldn't be too difficult to iron out. I also noticed that the `Download Firefox` inline link at the bottom of the 2 new pages should probably not be seen on Firefox browsers? Would be good to talk to the stakeholders + Adria about this?

Otherwise, good work! I'll approve it once things are patched up

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html (hunk 2 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:11]: Adria/Leonie approved the copy changes :)

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html (hunk 5 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:20]: Adria/Leonie approved the copy changes :)

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:1]: Oh, right, of course they're linking to each other. But I'll ask about sub nav and features page.

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html (hunk 6 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:59]: ✅

File: bedrock/firefox/templates/firefox/features/pdf-free-fr.html (hunk 4 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-free-fr.html:71]: Yeah but they didn't give me one and so Adria said this was okay as is.

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:1]: Adria confirms no other links needed.

File: bedrock/firefox/templates/firefox/features/pdf-editor-fr.html (hunk 1 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-editor-fr.html:30]: ✅

File: bedrock/firefox/templates/firefox/features/pdf-editor-fr.html (hunk 3 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-editor-fr.html]: ✅

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html (hunk 7 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-complete-fr.html]: ✅

File: bedrock/firefox/templates/firefox/features/pdf-editor-fr.html (hunk 1 above)
Comment (reemhamz) [reply bedrock/firefox/templates/firefox/features/pdf-editor-fr.html:30]: **Nit:** This still needs to be separated

File: bedrock/firefox/templates/firefox/features/pdf-complete-fr.html [hunk 8]
```diff
@@ -85,10 +84,9 @@ <h2>Enregistrer sa déclaration au format PDF</h2>
...
 <p>Enregistrez tous vos PDF dans des dossiers bien nommés pour une gestion facile et sécurisée.</p>
 
-<hr>
-
-<h2>Vous n'avez pas encore Firefox ?</h2>
-<p><a href="{{ url('firefox.features.index') }}">En savoir plus</a> sur nos fonctionnalités, ou <a href="">Téléchargez Firefox</a> dès maintenant.</p>
-
+<div class="hide-from-firefox">
+  <h2>Vous n'avez pas encore Firefox ?</h2>
+  <p><a href="{{ url('firefox.features.index') }}" data-cta-link="Learn more about our features">En savoir plus</a> sur nos fonctionnalités, ou <a href="{{ url('firefox.new') }}" data-link-text="Download Firefox" data-link-position="content">Téléchargez Firefox</a> dès maintenant.</p>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-complete-fr.html:89]: **Question:** Does the first anchor tag in this line also need a `data-link-position`?

File: bedrock/firefox/templates/firefox/features/pdf-editor-fr.html [hunk 9]
```diff
@@ -96,10 +95,9 @@ <h2>Modifier un PDF en ligne rapidement et facilement</h2>
...
 <p>Modifier des PDF oui, mais de façon simple et sécurisée !</p>
 
-<hr>
-
-<h2>Vous n'avez pas encore Firefox ?</h2>
-<p><a href="{{ url('firefox.features.index') }}">En savoir plus</a> sur nos fonctionnalités, ou <a href="">Téléchargez Firefox</a> dès maintenant.</p>
-
+<div class="hide-from-firefox">
+  <h2>Vous n'avez pas encore Firefox ?</h2>
+  <p><a href="{{ url('firefox.features.index') }}" data-cta-link="Learn more about our features">En savoir plus</a> sur nos fonctionnalités, ou <a href="{{ url('firefox.new') }}" data-link-text="Download Firefox" data-link-position="content">Téléchargez Firefox</a> dès maintenant.</p>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-editor-fr.html:100]: **Question:** Does the first anchor tag in this line also need a `data-link-position`?

File: bedrock/firefox/templates/firefox/features/pdf-free-fr.html [hunk 10]
```diff
@@ -83,6 +83,6 @@ <h2>Adoptez la simplicité et la sécurité avec l'éditeur PDF Firefox</h2>
 
 <p>Découvrez comment Firefox peut rendre vos tâches quotidiennes plus simples et plus sécurisées.</p>
 
-<p>Téléchargez Firefox dès aujourd'hui et reprenez le contrôle de vos documents PDF&nbsp;!</p>
+<p><a href="{{ url('firefox.new') }}" data-link-text="Download Firefox" data-link-position="content">Téléchargez Firefox</a> dès aujourd'hui et reprenez le contrôle de vos documents PDF&nbsp;!</p>
```
Comment (reemhamz) [inline bedrock/firefox/templates/firefox/features/pdf-free-fr.html:86]: **Nit:** We should also probably hide this line from Firefox browsers

**Review by reemhamz — APPROVED** [review]
Just a couple more nits but should be an easy fix. 
Good job again! r+wc 💯

File: bedrock/firefox/templates/firefox/features/pdf-free-fr.html (hunk 10 above)
Comment (stephaniehobson) [inline bedrock/firefox/templates/firefox/features/pdf-free-fr.html:86]:
```suggestion
<p class="hide-from-firefox"><a href="{{ url('firefox.new') }}" data-link-text="Download Firefox" data-link-position="content">Téléchargez Firefox</a> dès aujourd'hui et reprenez le contrôle de vos documents PDF&nbsp;!</p>
```

File: bedrock/firefox/templates/firefox/features/pdf-editor-fr.html (hunk 9 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-editor-fr.html:100]: No, there's nothing else to confuse this one with.

File: bedrock/firefox/templates/firefox/features/pdf-free-fr.html (hunk 10 above)
Comment (stephaniehobson) [reply bedrock/firefox/templates/firefox/features/pdf-free-fr.html]: ✅

**Review by NameYourOwn — APPROVED** [review]

**Pull request merged**